swagger/operation.go

```diff
@@ -28,12 +38,35 @@ func Interface(s *spec.Swagger, op *spec.Operation) string {
 		input = fmt.Sprintf("i *models.%sInput", capOpID)
 	}
 
-	successType := SuccessType(s, op)
-	if successType == nil {
-		return fmt.Sprintf("%s(ctx context.Context, %s) error", capOpID, input)
+	includeNames := false
+	returnTypes := []string{}
+	returnTypesWithNames := []string{}
```

Comment:
If consistency is the most important thing here, it probably makes sense to just do it in a comment.